outputs/bench.go

```diff
@@ -0,0 +1,53 @@
+package outputs
+
+import (
+	"fmt"
+	"io"
+	"time"
+
+	"github.com/goss-org/goss/resource"
+	"github.com/goss-org/goss/util"
+)
+
+type Bench struct{}
```

Comment:
Mentioned in docs, also mentioned it may not be stable in future releases.. Was using it for debugging and got tired of it not being part of goss =)

Logging may replace it, and it may get deleted in the future.

Thinking through it, part of me wonders if maybe I should remove it from this PR. Leaning towards that, but will sleep on it.

Reply:
🤷 . I'd say leave it in if it was useful.

Reply:
Slept on it, removing it for now. Initial logging support was added as part of PR a few months back. I have some thoughts on expanding logging to be a lot more useful. Once that story concludes, perhaps the bench output will not be useful.

I'll open up some issues after this PR is merged.